api_tests/base/test_sloan_study.py

```diff
@@ -269,3 +269,8 @@ def test_user_override_cookie_false(self, app, user, preprint):
         cookies = resp.headers.getall('Set-Cookie')
 
         assert f' dwf_{SLOAN_COI_DISPLAY}=False; Domain=.osf.io; Path=/; samesite=None; Secure' in cookies
+
+    def test_browseable_api(self, app):
+        headers = {'accept': 'application/vnd.api+json;version=latest'}
```

Comment:
```suggestion
        headers = {'accept': 'html/text'}
```

Reply:
Or you could add two more requests, one with that, and one with the format query parameter. But the jsonapi accept header should just return a json blob rather than the browsable api response, which I don't believe would replicate the 502 error.